simbad/command_line/__init__.py

```diff
@@ -588,6 +590,55 @@ def get_work_dir(run_dir, work_dir=None, ccp4_jobid=None, ccp4i2_xml=None):
     return os.path.abspath(work_dir)
 
 
+def make_output_dir(run_dir, output_dir, csv, mr_program):
+    """Make a directory containing output files
+
+    Parameters
+    ----------
+    run_dir : str
+        The path to the run directory
+    output_dir : str
+        The path to the output directory to create
+    csv : file
+        CSV file containing results
+    mr_program : str
+        The MR program used
+    """
+    import pandas as pd
+    import shutil
+    df = pd.read_csv(csv)
+    data = df.pdb_code.tolist()
+
+    if not os.path.isdir(output_dir):
+        os.mkdir(output_dir)
+
+    for pdb_code in data:
+        pdb_output_path = os.path.join(output_dir, pdb_code)
+        if not os.path.isdir(pdb_output_path):
+            os.mkdir(pdb_output_path)
+            mr_workdir = os.path.join(
+                run_dir, 'mr_search', pdb_code, 'mr', mr_program)
+            shutil.copyfile(os.path.join(
```

Comment:
I think this is terribly difficult to read/follow ... I would try something along the following lines.

```python
files_to_copy = [
    os.path.join(mr_workdir, '{0}_mr.log'.format(pdb_code)),
    os.path.join(mr_workdir, 'refine', '{0}_refinement_output.pdb'.format(pdb_code)),
    os.path.join(mr_workdir, 'refine', '{0}_refinement_output.mtz'.format(pdb_code)),
    os.path.join(mr_workdir, 'refine', '{0}_ref.log'.format(pdb_code)),
    os.path.join(mr_workdir, 'refine', '{0}_refmac_2fofcwt.map'.format(pdb_code)),
    os.path.join(mr_workdir, 'refine', '{0}_refmac_fofcwt.map'.format(pdb_code)),
]
for f in files_to_copy:
    shutil.copy(f, pdb_output_path)
```
The [`copy`](https://docs.python.org/2/library/shutil.html#shutil.copy) function allows to specify a target directory.